examples/demo.md

```diff
@@ -12,6 +12,9 @@ An introduction slide can be defined by using a front matter at the beginning of
 ---
 title: My presentation title
 sub_title: An optional subtitle
+event: The event the presentation is for
```

Comment:
Same here, this will cause the demo presentation to no longer work with the latest version so I'd rather keep it hidden until the next release.